scaleout/stackn/values.yaml

```diff
@@ -1,6 +1,96 @@
-# Default values for alliance-chart.
 # This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
+# Declare variables to be passed into STACKn templates.
+
+# REQUIREMENT: 
+# - set a storage class with ability to serve ReadWriteMany
+#   Name: storageClassName, and/or set anchor &śtorage_class
+#   Description: Set a storage class for the resources that are reused for multi-mount-points in cluster. To reduce wasteful copying we allow to use the same dataset volume to be mounted multiple times.
+#   Default: microk8s-hostpath, use nfs-client for docker-for-desktop
+# - replace <your-domain.com>, search and replace 
+# - cluster_config , kubernetes cluster
+
+#NOTES
+# - For local development/testing consider setting "oidc.verify_ssl" to false 
+
+#node anchors used to reduce manual configuration
+global:
+  db:
+    name: &db_name stackn
+    user: &db_user stackn
+    password: &db_password fdfweorkpokf931d0vsdf9
```

Comment:
Password in this line?

Reply:
Thank you Morgan for noticing these lines!